### ISSUE TITLE
Why not install binarytorch/larecipe as a dependency ?

### ISSUE THREAD

**Issue by Tantrisse** [icodestuff-io/ladocumenter#1]
Hi, this is more of a interrogation than a real issue but why do we have to install larecipe manually instead of having it installed as a dependency by ladocumenter ?

If larecipe, one day, update any of the files used on this package it may break ladocumenter.
Having larecipe as a dependency of ladocumenter will lock the version of larecipe installed to something you are sure is working with ladocumenter.

What do you think about it ?

**Comment by Solomon04** [comment]
Good idea. I'll test it out this weekend, and if everything works accordingly, I'll add it as a project level dependency.
